### PR TITLE
fix(#1507): remove showcount prop from textarea

### PR DESCRIPTION
**Pull request by chrisolsen**

### CODE REVIEW

**Review by ArakTaiRoth — DISMISSED** [review]

File: libs/web-components/src/components/comp-test.svelte [hunk 1]
```diff
@@ -0,0 +1,24 @@
+<svelte:options tag="goa-comp-test" />
```
[the remark this answers is not in the document]
Comment (chrisolsen) [reply libs/web-components/src/components/comp-test.svelte]: Yes :)

**Pull request merged**